docs/formats.rst

```diff
@@ -735,13 +735,13 @@ Example file:
 .. versionadded:: 2.3
 
 .NET Resource (.resx) file is a monolingual XML file format used in Microsoft
-.NET Applications. It works with .resw files as well as they use identical
+.NET applications. It works with .resw files as well as when they use identical
```

Comment:
What works?

Reply:
@nijel

Reply:
My question is concerning what "it" is here. I am unsure what works in the first place.

Reply:
AFAIK the only real difference is extension.

Looking at https://lingohub.com/developers/resource-files/resw-resx-localization:

> These 2 file types have different extensions (.resw, .resx) but the content is the same. The only difference is in the usage of these file types in your application development process.

Reply:
Was just the wording that had me diddled.